diagnostic.md

```diff
@@ -6,21 +6,21 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    You can define and specify the new url route and template in the router, which will create other objects.
```

Comment:
You are close.  You are right that the Ember Application Router defines the route.  We also need to use the Ember Route to specify how to load model data using the model hook.